### ISSUE TITLE
Processing dots

### ISSUE THREAD

**Issue by fredkgit** [Orange-OpenSource/Cloudnet-TOSCA-toolbox#10]
Processing  ".."  or "..." (as per yaml 1.2) which are often used in NFV examples. The program would display the yaml error instead of crashing (improper return code?),  either as a warning an then continue or as an error and then terminate.

**Comment by philippemerle** [comment]
Could you provide a short example illustrating the issue?

**Comment by fredkgit** [comment]
[trace_issue02.txt](https://github.com/Orange-OpenSource/Cloudnet-TOSCA-toolbox/files/5601584/trace_issue02.txt)

**Comment by fredkgit** [comment]
Actually I got the result in the attached doc and write the issue several months ago. I just try again, and got this : Translate sunshinedbsimple.vnfd.tosca.yaml...
[ERROR] sunshinedbsimple.vnfd.tosca.yaml: mapping values are not allowed here
  in "sunshinedbsimple.vnfd.tosca.yaml", line 41, column 17
So it seems it has been fixed or come from some other reason. Maybe we cann close the issue and I'll open it in case I see this again?
